Simulator/components/SimulatorAttributes.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component, PropTypes } from 'react';
+import { FormGroup, FormControl, ControlLabel } from 'react-bootstrap';
+
+class SimulatorAttributes extends Component {
+  constructor(props) {
+    super(props);
+    this.handleAttrChange = this.handleAttrChange.bind(this);
+  }
+
+  handleAttrChange(e) {
+    const newAttrs = { ...this.props.wdcAttrs, [e.target.id]: e.target.value };
+    this.props.setWdcAttrs(newAttrs);
+  }
+
+  render() {
+    return (
+      <div className="data-gather-properties">
+        <FormGroup>
+          <h2> Web Data Connector Properties </h2>
+          <ControlLabel> Connection Name </ControlLabel>
+          <FormControl
+            type="text"
+            disabled={this.props.disabled}
+            id="connectionName"
+            value={this.props.wdcAttrs.connectionName}
+            onChange={this.handleAttrChange}
+          />
+          <ControlLabel> Connection Data </ControlLabel>
+          <FormControl
+            type="textarea"
+            disabled={this.props.disabled}
+            id="connectionData"
+            value={this.props.wdcAttrs.connectionData}
+            onChange={this.handleAttrChange}
+          />
+          <ControlLabel> Username </ControlLabel>
+          <FormControl
+            type="text"
+            disabled={this.props.disabled}
+            label="Username"
+            id="username"
+            value={this.props.wdcAttrs.username}
+            onChange={this.handleAttrChange}
+          />
+          <ControlLabel> Password </ControlLabel>
+          <FormControl
+            type="text"
```

Comment:
Would it make sense to switch this to be a password field? I know it never has been, but it seems weird, especially for demos, that we present the password so prominently